sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -49,8 +49,11 @@ private class CodeFormatter {
   private var currentLine = 1
 
   private def addLine(line: String): Unit = {
-    val indentChange =
+    val indentChange = if (line.startsWith("/*") || line.startsWith("//")) {
```

Comment:
I've updated the PR to support basic multi-line comments and added relevant tests. We can probably do fancier things if we'd like to support inline comments (`// hello world /*`) or nested comments, but that's probably not needed here (and should ideally require a standard parser)